examples/stocks/lib/stock_symbol_viewer.dart

```diff
@@ -102,7 +107,10 @@ class StockSymbolBottomSheet extends StatelessWidget {
       decoration: new BoxDecoration(
         border: new Border(top: new BorderSide(color: Colors.black26))
       ),
-      child: new StockSymbolView(stock: stock)
+      child: new _StockSymbolView(
+        stock: stock,
+        arrow: new StockArrow(percentChange: stock.percentChange)
```

Comment:
oh, i see... yeah i guess it makes sense to create the hero outside then. that makes me a bit sad but *shrug*